### PR TITLE
Release 1.6 test

### PR DESCRIPTION
**Pull request by vivekbhr**
Just some spelling mistake and readability issue corrections with the three tools that I am looking into..

### CODE REVIEW

File: bin/bigwigCorrelate [hunk 1]
```diff
@@ -124,15 +124,15 @@ def bigwigCorrelateArgs(case='bins'):
 
     optional.add_argument('--chromosomesToSkip',
                         metavar='chr1 chr2',
-                        help='List of chromosomes that do not want to be included '
-                             'for the correlation. Useful to remove "ramdom" or "extra" chr.',
+                        help='List of chromosomes that you do not want to be included '
```
Comment (bgruening) [inline bin/bigwigCorrelate]: `List of chromosomes to exclude for the correlation.`

File: deeptools/_version.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 # This file is originally generated from Git information by running 'setup.py
 # version'. Distribution tarballs contain a pre-generated copy of this file.
 
-__version__ = '1.6.0'
+__version__ = '1.6.0-173-gc811461'
```
Comment (bgruening) [inline deeptools/_version.py]: Is this necessary?

Comment (dpryan79) [inline deeptools/_version.py]: I would hope not. Truth be told I think it's going to be `__version__ = '2.0.0'` anyway.

Comment (fidelram) [inline deeptools/_version.py]: This is how it works: whenever you run setup.py the version number is automatically updated to match the commit hash. This is only relevant for the installation and should not be push to the repository.

Comment (dpryan79) [inline deeptools/_version.py]: Maybe this should be in `.gitignore` then.

Comment (fidelram) [inline deeptools/_version.py]: I just did that. Still not sure what will happen when I want to commit a
change on that file

On Thu, Dec 3, 2015 at 9:21 AM, Devon Ryan <notifications@github.com> wrote:

> In deeptools/_version.py
> <https://github.com/fidelram/deepTools/pull/118#discussion_r46523567>:
>
> > @@ -2,4 +2,4 @@
> >  # This file is originally generated from Git information by running 'setup.py
> >  # version'. Distribution tarballs contain a pre-generated copy of this file.
> >
> > -__version__ = '1.6.0'
> > +__version__ = '1.6.0-173-gc811461'
>
> Maybe this should be in .gitignore then.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fidelram/deepTools/pull/118/files#r46523567>.
>



-- 

Fidel Ramirez

Comment (dpryan79) [inline deeptools/_version.py]: It'll get ignored, just `git commit` after `git rm deeptools/_version.py`

**Pull request merged**